tools/initial-setup.sh

```diff
@@ -38,6 +38,8 @@ cat > $SETUP_SCRIPT <<-EOF
 #!/bin/bash -xe
 apt-get -yqm update
 apt-get -yq upgrade python3.8
```

Comment:
Is `python3.8` needed in addition to `python3.8-dev`?
```suggestion
```

Reply:
I guess you're right, it's a dependency and would be installed automatically. But it doesn't hurt to have that line explicitly, though we can remove it.

Reply:
Thanks for the clarification. I agree it doesn't hurt to leave it there, so let's leave it as is.

Reply:
I had the same question here: https://github.com/NCBI-Codeathons/psss-team4/pull/3#discussion_r719040253

But yeah, we can just leave it for now.